Userland/Libraries/LibJS/Runtime/Temporal/AbstractOperations.cpp

```diff
@@ -1870,4 +1870,22 @@ ThrowCompletionOr<DifferenceSettings> get_difference_settings(VM& vm, Difference
     };
 }
 
+// https://tc39.es/proposal-temporal/#sec-tointegerifintegral
+ThrowCompletionOr<double> to_integer_if_integral(VM& vm, Value argument, StringView unit)
```

Comment:
The commit you linked is a plain rename, so do that instead?

Reply:
The function with the prior rename (ToIntegerWithoutRounding ) did not exist previously, so there was no function to rename.  If its preferable, I could make a commit adding the function under the old name and then rename it in a second commit.

Reply:
No idea what you're talkking about
https://github.com/SerenityOS/serenity/blob/f5a3355639a826d041b8b841e5525397677b42ee/Userland/Libraries/LibJS/Runtime/Temporal/AbstractOperations.h#L204-L221

Reply:
Huh, don't know how I missed that